tools/gpstest/README.md

```diff
@@ -1,38 +1,72 @@
-# GPS test setup
+# GPS Teststation Setup
+Testing the GPS receiver using GPS spoofing. At the moment only
+static location relpay is supported.
 
 # Usage
 ```
-# replaying a static location
-./gpstest.sh -e <ephemeris file> -s <static location>
+# on host, start gps signal simulation
+./run_static_lime.py
 
-# replaying a prerecorded route (NMEA cvs file)
-./gpstest.sh -e <ephemeris file> -d <dynamic location>
+# on device
+python -m unittest test_gps.py
 ```
 
-If `-e` is not provided the latest ephemeris file will be downloaded from
+`run_static_lime.py` downloads the latest ephemeris file from
 https://cddis.nasa.gov/archive/gnss/data/daily/20xx/brdc/.
-(TODO: add auto downloader)
 
-# Hardware Setup
 
+# Hardware Setup
 * [LimeSDR USB](https://wiki.myriadrf.org/LimeSDR-USB)
-* Asus AX58BT antenna
+* Asus AX58BT (antenna)
 
 # Software Setup
 * https://github.com/myriadrf/LimeSuite
-To communicate with LimeSDR the LimeSuite is needed it abstracts the direct
+
+Communicate with LimeSDR the LimeSuite is needed it abstracts the direct
 communication. It also contains examples for a quick start.
 
-The latest stable version (22.09) does not have the corresponding firmware
+The latest stable version(22.09) does not have the corresponding firmware
 download available at https://downloads.myriadrf.org/project/limesuite. Therefore
 version 20.10 was chosen.
 
+A successful build should gives similar to:
+```
+./LimeUtil --info
+######################################################
+## LimeSuite information summary
+######################################################
```

Comment:
I removed a bunch of this stuff from the initial PR, and looks those changes didn't get merged properly.

Reply:
oh, I misread that